File: LLM-Translations/GPT-4/codenet/C/Java/s383338432.java
public class s383338432 {
    public static void main(String[] args) {
        java.util.Scanner scanner = new java.util.Scanner(System.in);
        int height = scanner.nextInt();
        int width = scanner.nextInt();
        scanner.nextLine(); // To consume the newline character
        String inputStr;
        
        for (int i = 0; i < width + 2; i++) {
            System.out.print("#");
        }
        System.out.println();
        
        for (int i = 0; i < height; i++) {
            inputStr = scanner.nextLine();
            System.out.println("#" + inputStr + "#");
        }
        
        for (int i = 0; i < width + 2; i++) {
            System.out.print("#");
        }
        System.out.println();
        
        scanner.close();
    }
}
// 

File: LLM-Translations/GPT-4/codenet/C/Go/s383338432.go
package main

import (
	"fmt"
)

func main() {
	var height, width int
	fmt.Scanf("%d %d", &height, &width)
	inputStr := make([]string, height)

	for i := 0; i < width+2; i++ {
		fmt.Print("#")
	}
	fmt.Println()

	for i := 0; i < height; i++ {
		fmt.Scanf("%s", &inputStr[i])
		fmt.Printf("#%s#\n", inputStr[i])
	}

	for i := 0; i < width+2; i++ {
		fmt.Print("#")
	}
	fmt.Println()
}

// 
